iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -768,4 +783,10 @@ private void addUserIdentitiesToRequest(UserRequest request, List<UserIdentity>
         }
     }
 
+    private boolean hasBundledSDK(EventProcessingRequest processingRequest) {
+        Map<String, String> integrationAttributes = processingRequest.getIntegrationAttributes();
+        return integrationAttributes != null &&
+                integrationAttributes.getOrDefault("sdkVersion", null) != null;
```

Comment:
this should be prefixed with Iterable, ex "Iterable.sdkVersion"

Reply:
Would `iterableSdkVersion` be ok, or is the convention to have namespaces like `Iterable.*`?

Reply:
Iterable.* is the convention

Reply:
Fixed